frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -170,6 +170,35 @@ const TAGGED_TEMPLATES = {
       </>
     );
   },
+  editedMacosMinVersion: (activity: IActivity) => {
+    const editedActivity =
+      activity.details?.minimum_version === "" ? "removed" : "updated";
+
+    const versionSection = activity.details?.minimum_version ? (
+      <>
+        to version <b>{activity.details.minimum_version}</b>
```

Comment:
to match the spec in the Figma file

```suggestion
        to <b>{activity.details.minimum_version}</b>
```